### DISCUSSION TITLE
Custom Avatar's hands are glitching when using VR controllers (model exported from Blender 2.83 using blender hubs exporter)

### DISCUSSION THREAD

**Discussion by Rawdreen** [mozilla/hubs#2827] [General Questions & Discussion] [answered]
**DESCRIPTION:**
I made a custom model for Hubs and using proper [Avatar export pipeline](https://github.com/MozillaReality/hubs-avatar-pipelines/tree/master/Blender/AvatarBot) exported it to Hubs with .glb format. 

**loop and morph animations (like eyes and lips) are working well but when the VR hand controller buttons (sometimes even if buttons are not pressed) are pressed hands are glitching.** 

[Mozilla Mixed Reality youtube channel custom Avatar tutorials ](https://www.youtube.com/playlist?list=PLCxaiaRxTL6-3pxUsfSa7lzKruGu5UEat) had not covered the hand controller part. 

**TRIED TO RESOLVE: **
I've tried to use different blender version (2.80 - 2.83)s and export settings. None

**POSSIBLE SOLUTION**
I think I'm missing some export settings or I am messing up something when installing bones to my custom avatar from an avatar provided by [Mozilla Hubs](https://github.com/MozillaReality/hubs-avatar-pipelines/tree/master/Blender/AvatarBot).

an example of a glitch: https://youtu.be/fM4LpUmbvds

**Comment by brianpeiris** [comment, marked as answer]
Hi @Rawdreen. That's an odd issue. Do the hands still glitch if you hold your head still? I wonder if it's some sort of culling issue, though it is more likely to be an issue with the bones. It might help if you're able to share the particular avatar GLB with us, so that we can do some debugging ourselves.

**Reply by emclaren** [reply]
I'm going to mark this as answered because we haven't heard back in a few weeks - but @Rawdreen feel free to reach out if you want us to look at this again

**Reply by Rawdreen** [reply]
Hi, there (Sorry for my long absence)!
We've figured out the issue with a glitch, here is how we fix it in Blender:

1. We turned off all checkboxes in the Animation window.
![image](https://user-images.githubusercontent.com/23343509/92165556-aae2b500-ee50-11ea-9804-00ddc798bee7.png)

2. We disabled the "Always Sample Animations" checkbox in the export settings of Blender.
![image](https://user-images.githubusercontent.com/23343509/92165649-cd74ce00-ee50-11ea-8f08-c9fd0a5b7efb.png)


After loading it into Mozilla Hubs everything worked properly! 😊 


P.S.: The original solution was proposed by Jim in Discord's Mozilla Hubs channel.  (Sorry, couldn't find his GitHub nickname )

**Reply by emclaren** [reply]
Thanks for reporting back! this will be super helpful to other folks
